olive/systems/azureml/aml_system.py

```diff
@@ -405,6 +476,27 @@ def pass_runner_pipeline():
 
         return pass_runner_pipeline()
 
+    def _run_workflow_job(
```

Comment:
nit: maybe we can generalize the `_run_job` method to make the job streaming and output download optional? The job submission logic is the same.

Reply:
Sounds good. Updated.